docs/Concepts/Merge.md

```diff
@@ -22,29 +25,36 @@ Execution and consensus clients communicate with each other using the
 
 ### Execution clients
 
-Execution clients, such as [Besu], manage the state and execute transactions on the execution layer.
+Execution clients, such as [Besu](https://besu.hyperledger.org/en/stable/), manage the state and execute transactions on
+the execution layer.
```

Comment:
are we referring to the `world state`? consensus clients also have a state and it's very different... @ajsutton you might be a good person to ask?

Reply:
Yeah, the execution client manages the world state for the execution layer.  I'd be tempted to write it as
```suggestion
Execution clients, such as [Besu](https://besu.hyperledger.org/en/stable/), manage the execution layer, including executing transactions and updating the world state.
```